OsuSharp/Endpoints/Beatmaps.cs

```diff
@@ -155,10 +155,54 @@ public async Task<BeatmapExtended[]> GetBeatmapsAsync(params int[] ids)
   /// <a href="https://osu.ppy.sh/docs/index.html#get-beatmap"/>
   /// </summary>
   /// <param name="id">The ID of the beatmap.</param>
+  /// <param name="ruleset">The ruleset to get the attributes from.</param>
   /// <returns>The difficulty attributes or null, if the beatmap was not found.</returns>
-  public async Task<DifficultyAttributes?> GetDifficultyAttributesAsync(int id)
+  public Task<DifficultyAttributes?> GetDifficultyAttributesAsync(int id, string? ruleset = null) => GetDifficultyAttributesInternalAsync(id, null, ruleset);
```

Comment:
```suggestion
  public Task<DifficultyAttributes?> GetDifficultyAttributesAsync(int id, Ruleset? ruleset = Ruleset.Osu) => GetDifficultyAttributesInternalAsync(id, null, ruleset);
```
We can make use of the ruleset enum here as the query string builder automatically takes care of parsing it.

Reply:
My bad, I didn't even check if this existed.